### DISCUSSION TITLE
[메일] 정찰 보상 획득 시 상점 메일 탭으로 분류되는 현상

### DISCUSSION THREAD

**Discussion by QgamesGeonPyoNoh** [planetarium/NineChronicles#5884] [General]
### APV version info

v200160

### Client version info

v160.1.1

### Device

Windows, Android, iOS

### detailed device info

Windows 10, Android 12, iOS 17

### account address info

0x3146FE0E47A1998e14cd8a33640e3648A5A90276

### What happened?

[내용]
정찰 보상 획득 시 상점 메일 탭으로 분류되는 현상
-PC / Android / iOS 환경에서 확인하였습니다.
-첨부한 스크린 샷 확인 부탁드립니다.

![Image](https://github.com/planetarium/NineChronicles/assets/150774808/09dceb66-6f6b-4375-90df-218507c1852c)

### How to reproduce the issue

[재현스텝]
1. 메인 로비 - 정찰 보상 획득 진행
2. 메일 - 상점 버튼 클릭
3. 정찰 보상 획득 시 상점 메일 탭으로 분류되는 현상 확인

[Should be]
정찰 보상 획득 시 시스템 메일 탭으로 분류되어야 합니다.

**Comment by Namyujeong** [comment]
시즌패스
정찰보상
몬스터콜렉션

3개 보상은 시스템 카테고리에 들어가야 하는데 관습적으로 마켓 카테고리를 계속 쓰고 있었던 것 같습니다.
시스템 카테고리를 여태 한번도 안 썼는데, 제대로 분류되면 좋겠네요.

**Comment by ipdae** [comment]
타입논의가 정해지고 진행하면 될것같습니다.
